src/views/admin/PendingAccounts.vue

```diff
@@ -87,7 +87,11 @@
     },
     async updatePendingAccount(){
       this.isLoading = true
-      await this.$store.dispatch('fetchPendingUserAccounts')
+      try {
+        await this.$store.dispatch('fetchPendingUserAccounts')
+      } catch (e:any) {
+        console.error('Failed to fetch pending accounts', e.message + "\nDebug: " + JSON.stringify(e))
+      }
```

Comment:
I was expecting a message for the user in lieu of the "no pending accounts". Here the console.error doesn't bring a lot of difference than not catching the exception, does it ?

Reply:
This commit fixes the error by catching it and reporting it in the console. The next commit adds functionality by providing feedback to the user.

Reply:
Squashed commits. Changed.